containerize/Containerfile

```diff
@@ -5,34 +5,61 @@
 # Prepare dependencies
 FROM registry.access.redhat.com/ubi9-minimal AS deps
 
+# Indicate if dependencies were prefetched using Cachi2
+# They mush be located at /cachi2/output/deps
+ARG PREFETCH=false
+
+# These versions should be consistent with the listed in the artifacts.lock.yaml file
+ARG ZAP_VERSION=2.15.0
+ARG FF_VERSION=128.6.0esr
```

Comment:
Firefox is now locked to a specific version. Previously, it was downloading the latest one, but in order to validate the checksum with Cachi2, we need to specify a fixed version. We could continue fetching the latest version when binaries aren’t pre-fetched, but I wanted to maintain consistency across both approaches.

Reply:
Sounds good to me